src/common/snippets/src/lowered/loop_manager.cpp

```diff
@@ -438,6 +463,31 @@ void LinearIR::LoopManager::fuse_loops(LinearIR::constExprIt loop_begin_target,
     remove_loop_info(from);
 }
 
+std::vector<lowered::pass::PassPipeline> LinearIR::LoopManager::fuse_loop_handlers(
+    std::vector<lowered::pass::PassPipeline>& from,
+    std::vector<lowered::pass::PassPipeline>& to) {
```

Comment:
Minor (just for the better understanding):
```suggestion
    std::vector<lowered::pass::PassPipeline>& lhs,
    std::vector<lowered::pass::PassPipeline>& rhs) {
```